SPECIFICATIONS.md

```diff
@@ -170,7 +174,7 @@ The logic language is descibed by the following EBNF grammar:
 <check> ::= "check" <sp> "if" <sp> <rule_body> (<sp>? " or " <sp>? <rule_body>)* <sp>?
 <policy> ::= ("allow" | "deny") <sp> "if" <sp> <rule_body> (<sp>? " or " <sp>? <rule_body>)* <sp>?
 
-<rule_body> ::= <rule_body_element> <sp>? ("," <sp>? <rule_body_element> <sp>?)*
+<rule_body> ::= <rule_body_element> <sp>? ("," <sp>? <rule_body_element> <sp>?)* (<sp> <origin_clause>)?
```

Comment:
missing `@` token?

Reply:
I chose to use `trusting` everywhere, for consistency and readability